packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -1045,6 +1061,27 @@ class _CupertinoDatePickerDateTimeState extends State<CupertinoDatePicker> {
     );
   }
 
+  // Builds the time seperator column.
+  Widget _buildTimeSeperatorWidget(
+    double offAxisFraction,
+    TransitionBuilder itemPositioningBuilder,
+    Widget? selectionOverlay,
+  ) {
+    return CupertinoPicker(
```

Comment:
Should wrap this with an ExcludeSemantics, so it's not picked up accessibility readers. It also feels odd that this is a CupertinoPicker with only one option. Can this be a non-picker widget?

Reply:
Since the time seperator widget uses a text field if we remove the cupertino picker widget wrapper we'll have this output (under the hood the time seperator column is taking the full height of the cupertino picker)
so for now I've added the `ExcludeSemantics` and waiting for your confirmation on this 
 
<img src="https://github.com/user-attachments/assets/53504226-1eea-4019-aafe-e8adc9f4f825" width="200"/>

Reply:
Lets just use the picker widget and makes things easier by picking up the existing styling and layout work. Otherwise it's probably a lot of work just for the code to feel better. Thank you for including the `ExcludeSemantics` widget.